web/vite.config.ts

```diff
@@ -28,4 +28,7 @@ export default defineConfig({
       "@/": `${resolve(__dirname, "src")}/`,
     },
   },
+  define: {
```

Comment:
Why do we need to define this?

Reply:
I'm getting an error like this: `Uncaught ReferenceError: global is not defined` https://github.com/vitejs/vite/issues/2778

![Screen Shot 2022-09-16 at 23 24 23](https://user-images.githubusercontent.com/110044923/190685503-d60f1981-0706-4cc6-a3aa-3fdcefda19ba.png)
I have never used vite.js and I used a solution I found on google.